resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -201,6 +209,25 @@ private[spark] class KubernetesClusterSchedulerBackend(
       sc.env.securityManager.isAuthenticationEnabled(),
       sc.env.securityManager.isSaslEncryptionEnabled())
   }
+  private var workingWithSparkJobResource =
+    conf.get("spark.kubernetes.jobResourceSet", "false").toBoolean
+
+  private val jobResourceName = conf.get("spark.kubernetes.jobResourceName", "")
+  private val appropriateHttpClientToken = {
+    conf.get(KUBERNETES_DRIVER_MOUNTED_OAUTH_TOKEN) match {
+      case Some(tokenPath) =>
+        Option.apply(Files.toString(new File(tokenPath), Charsets.UTF_8))
+      case None =>
+        Option.apply(Files.toString(new File(
+          Config.KUBERNETES_SERVICE_ACCOUNT_TOKEN_PATH), Charsets.UTF_8))
+    }
+  }
+  private val sparkJobResourceController =
+    if (conf.get("spark.kubernetes.jobResourceSet", "false").toBoolean) {
```

Comment:
this looks the same as `workingWithSparkJobResource` above -- can you use that instead of having the same `false` default value here that needs to be in sync?